### PR TITLE
Deploying on a node and no i-net connection. #12

### PR DESCRIPTION
**Pull request by BobVanB**
Creating a package_base for packages.

### CODE REVIEW

File: manifests/install.pp [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
   exec {
     'Download Grafana':
-      command => "curl -s -L http://grafanarel.s3.amazonaws.com/grafana-${::grafana::version}.tar.gz | tar xz",
+      command => "curl -s -L ${::grafana:package_base}/grafana-${::grafana::version}.tar.gz | tar xz",
```
Comment (dwerder) [inline manifests/install.pp]: As the CI mentions, there is a syntax error. Or better be said: there should be two ::

**Comment by dwerder** [comment]
everything else looks fine

**Comment by BobVanB** [comment]
Do i need to create a new push request?

**Comment by dwerder** [comment]
No. Just fix it in your branch.

**Pull request merged**